src/common.js

```diff
@@ -111,6 +115,7 @@ exports.ShellString = ShellString;
 
 // Return the home directory in a platform-agnostic way, with consideration for
 // older versions of node
+// XXX: I don't think this should be a part of plugin.utils. Not sure what should happen to it. Maybe just inline it.
```

Comment:
I'm not sure where this is used, or why it's needed. I'm thinking we should maybe just use an external ponyfill. (I like [this one](https://github.com/sindresorhus/home-or-tmp), because it works if the home dir doesn't exist. If we don't want that, [this one](https://www.npmjs.com/package/os-homedir) looks good, as does [this one](https://www.npmjs.com/package/real-homedir).)